test/puppetlabs/puppetdb/integration/sensitive_params.clj

```diff
@@ -0,0 +1,77 @@
+(ns puppetlabs.puppetdb.integration.sensitive-params
+  (:require
+   [clojure.java.shell :refer [sh]]
+   [clojure.string :as str]
+   [clojure.test :refer :all]
+   [puppetlabs.puppetdb.integration.fixtures :as int])
+  (:import
+   [java.net URI]))
+
+(defn pg-dump
+  "Returns the content of pg as a string via pg_dump."
+  [pg]
+  (let [db-config (:db-config pg)
+        db-uri (URI. (:subname db-config))
+        result (sh "pg_dump"
+                   "--username" (:user db-config)
+                   "--host" (.getHost db-uri)
+                   "--port" (str (.getPort db-uri))
+                   "--dbname" (subs (.getPath db-uri) 1)
+                   "--data-only"
+                   "--no-owner"
+                   "--encoding" "utf-8"
+                   "--format" "plain"
+                   :out-enc "utf-8")]
+    (when-not (zero? (:exit result))
+      (is (= 0 (:exit result)))
+      (binding [*out* *err*]
```

Comment:
Hmm, not sure that one's mine, but regardless, I didn't want to add an extra newline onto the subproc's stderr.